plugins/CircuitViewer/io/CircuitLoader.cpp

```diff
@@ -117,6 +114,107 @@ std::string _getMeshFilenameFromGID(const uint64_t gid,
+
+    // Old nomemclature
+    radii["NGC"] = 7.46;
+    radii["CRC"] = 7.28;
+    radii["BP"] = 6.74;
+    radii["L6CTPC"] = 7.00;
+    radii["ChC"] = 8.01;
+    radii["L4PC"] = 7.84;
+    radii["L2PC"] = 8.36;
+    radii["LBC"] = 9.28;
+    radii["NBC"] = 8.61;
+    radii["L5STPC"] = 8.61;
+    radii["L6CCPC"] = 7.71;
+    radii["L3PC"] = 7.70;
+    radii["ADC"] = 7.72;
+    radii["L5TTPC"] = 11.17;
+    radii["AHC"] = 7.54;
+    radii["L5UTPC"] = 9.75;
+    radii["MC"] = 9.37;
+    radii["DBC"] = 6.58;
+    radii["SBC"] = 7.75;
+    radii["L4SS"] = 8.87;
+    radii["BTC"] = 9.15;
+    radii["L4SP"] = 7.76;
+    radii["L6CLPC"] = 7.32;
+
+    // New nomemclature
+    radii["L1_DAC"] = 7.77242212296;
+    radii["L1_NGC-DA"] = 7.71745554606;
+    radii["L1_NGC-SA"] = 7.53378756841;
+    radii["L1_HAC"] = 7.48774570227;
+    radii["L1_DLAC"] = 7.63075244427;
+    radii["L1_SLAC"] = 6.54029955183;
+    radii["L23_PC"] = 7.7618568695;
+    radii["L23_MC"] = 8.25832914484;
+    radii["L23_BTC"] = 9.30396906535;
+    radii["L23_DBC"] = 6.67491123753;
+    radii["L23_BP"] = 6.47212839127;
+    radii["L23_NGC"] = 8.40507149696;
+    radii["L23_LBC"] = 9.0722948432;
+    radii["L23_NBC"] = 8.64244471205;
+    radii["L23_SBC"] = 7.77334165573;
+    radii["L23_ChC"] = 8.38753128052;
+    radii["L4_PC"] = 7.5360350558;
+    radii["L4_SP"] = 7.97097187241;
+    radii["L4_SS"] = 8.95526583989;
+    radii["L4_MC"] = 9.64632482529;
+    radii["L4_BTC"] = 8.13213920593;
+    radii["L4_DBC"] = 9.06448200771;
+    radii["L4_BP"] = 5.80337953568;
+    radii["L4_NGC"] = 7.61350679398;
+    radii["L4_LBC"] = 8.92902739843;
+    radii["L4_NBC"] = 9.02906776877;
+    radii["L4_SBC"] = 9.16592723673;
+    radii["L4_ChC"] = 8.54107379913;
+    radii["L5_TTPC1"] = 12.4378146535;
+    radii["L5_TTPC2"] = 12.9319117383;
+    radii["L5_UTPC"] = 7.55000627041;
+    radii["L5_STPC"] = 8.84197418645;
+    radii["L5_MC"] = 9.02451186861;
+    radii["L5_BTC"] = 9.24175042372;
+    radii["L5_DBC"] = 8.92543775895;
+    radii["L5_BP"] = 5.98329114914;
+    radii["L5_NGC"] = 6.62666320801;
+    radii["L5_LBC"] = 10.0915957915;
+    radii["L5_NBC"] = 9.00336164747;
+    radii["L5_SBC"] = 10.0064823627;
+    radii["L5_ChC"] = 7.85296694438;
+    radii["L6_TPC_L1"] = 7.11837192363;
+    radii["L6_TPC_L4"] = 7.08293668837;
+    radii["L6_UTPC"] = 7.74583534818;
+    radii["L6_IPC"] = 7.71743569988;
+    radii["L6_BPC"] = 7.14034402714;
+    radii["L6_MC"] = 9.55710192858;
+    radii["L6_BTC"] = 9.97289562225;
+    radii["L6_DBC"] = 9.88600463867;
+    radii["L6_BP"] = 5.98329114914;
+    radii["L6_NGC"] = 6.62666320801;
+    radii["L6_LBC"] = 9.49501685154;
+    radii["L6_NBC"] = 9.33885664259;
+    radii["L6_SBC"] = 8.06322860718;
+    radii["L6_ChC"] = 8.15173133214;
+
+    const auto& mtypes = circuit.getMorphologyTypeNames();
+    std::vector<float> output;
+    for (auto mtype : mtypes)
+    {
+        auto iter = radii.find(mtype);
+        if (iter == radii.end())
```

Comment:
ideally that doesn't happen, log when it does?

Reply:
Yes, for mtypes not in the list above (e.g. hippocampus). Do you want to see a warning printed for every cell for which its mytpe is not found or just once per mtype?

Reply:
Only once at the end that there were mtypes (maybe list them there) that were not in this list.